packages/account/src/Components/file-uploader-container/file-uploader.jsx

```diff
@@ -78,14 +78,14 @@ class FileUploader extends React.PureComponent {
         if (!!file_error_message || document_file.length < 1) return 0;
 
         // File uploader instance connected to binary_socket
-        const uploader = new DocumentUploader({ connection: WS.getSocket() });
+        const uploader = new DocumentUploader({ connection: this.props.getSocket() });
```

Comment:
I'm confused, why is this needed to get passed in?

Reply:
Because we don't have `WS` initiated in account package if the user has never opens up "/account" page. `WS` will be undefined and connection woud not work. We have to pass it in, when we want to use the component. Now that you've mentioned it tho, this requires a double check to see if the /account/proof-of-address works.

Reply:
@reza-fs Perhaps we can have an index file (or the like) that runs some basic init functions in the `account` package regardless of what is imported? That way even though `FileUploaderContainer` is being imported from `@deriv/account` without the Account templates/view having ever loaded, it'd still run the basic initialisation functions. What do you think?

Reply:
IIRC, we inject WS when `app.jsx` component for account package is loaded (passed by `passthrough` props). 

If I understood you correctly, we should create an `index.js` file with the initialization (Passing WS and other required libraries to that component). This makes me concerned about not these: 

- If the file upload container is moved to `deriv-components` (which I think it should, it is being re-used in multiple places, then it should not care about `WS` at all, after all, it is just passing the socket to `DocumentUploader` right?
- When should we initialize? If we do it on `core`, and then not use the account signup flow, doesn't it add additional complexity to lazy-load this? Unless we hold off the initialization until account signup is open, in that case, I don't see much difference between passing the websocket or initializing it and let the component use it.